2022-02-10 16:02:42,937 INFO    MainThread:604429 [wandb_setup.py:_flush():71] setting env: {}
2022-02-10 16:02:42,937 INFO    MainThread:604429 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-10 16:02:42,937 INFO    MainThread:604429 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_160242-13fexu1m/logs/debug.log
2022-02-10 16:02:42,937 INFO    MainThread:604429 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_160242-13fexu1m/logs/debug-internal.log
2022-02-10 16:02:42,938 INFO    MainThread:604429 [wandb_init.py:init():404] calling init triggers
2022-02-10 16:02:42,938 INFO    MainThread:604429 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-10 16:02:42,938 INFO    MainThread:604429 [wandb_init.py:init():460] starting backend
2022-02-10 16:02:42,938 INFO    MainThread:604429 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-10 16:02:42,951 INFO    MainThread:604429 [backend.py:ensure_launched():216] starting backend process...
2022-02-10 16:02:42,960 INFO    MainThread:604429 [backend.py:ensure_launched():221] started backend process with pid: 604461
2022-02-10 16:02:42,961 INFO    MainThread:604429 [wandb_init.py:init():469] backend started and connected
2022-02-10 16:02:42,963 INFO    MainThread:604429 [wandb_init.py:init():533] updated telemetry
2022-02-10 16:02:42,979 INFO    MainThread:604429 [wandb_init.py:init():563] communicating current version
2022-02-10 16:02:43,672 INFO    MainThread:604429 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-10 16:02:43,672 INFO    MainThread:604429 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-10 16:02:43,807 INFO    MainThread:604429 [wandb_init.py:init():606] starting run threads in backend
2022-02-10 16:02:45,873 INFO    MainThread:604429 [wandb_run.py:_console_start():1810] atexit reg
2022-02-10 16:02:45,873 INFO    MainThread:604429 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-10 16:02:45,874 INFO    MainThread:604429 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-10 16:02:45,875 INFO    MainThread:604429 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-10 16:02:45,875 INFO    MainThread:604429 [wandb_init.py:init():633] run started, returning control to user process
2022-02-10 16:02:45,875 INFO    MainThread:604429 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-10 16:02:45,876 INFO    MainThread:604429 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-10 16:02:45,877 INFO    MainThread:604429 [wandb_run.py:_restore():1752] restore
2022-02-10 16:02:48,382 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2022-02-10 16:02:48,516 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2022-02-10 16:02:48,818 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11592
}

2022-02-10 16:02:48,926 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11592
}

2022-02-10 16:02:49,027 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,128 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,229 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,331 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,432 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,533 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,731 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}

2022-02-10 16:02:49,987 INFO    MainThread:604429 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11592
  total_bytes: 11592
}
local_info {
}

2022-02-10 16:02:51,120 INFO    MainThread:604429 [wandb_run.py:_append_files():2180] logging synced files
